libraries/AP_Notify/Buzzer.h

```diff
@@ -38,19 +36,16 @@ class Buzzer: public NotifyDevice
     /// on - turns the buzzer on or off
     void on(bool on_off);
 
-    // Patterns - how many beeps will be played
-    enum BuzzerPattern {
-        NONE = 0,
-        SINGLE_BUZZ = 1,
-        DOUBLE_BUZZ = 2,
-        GPS_GLITCH = 3, // not used
-        ARMING_BUZZ = 4,
-        BARO_GLITCH = 5,
-        EKF_BAD = 6
-    };
+    // Patterns - how many beeps will be played; read from
+    // left-to-right, each bit represents 10ms
+    const uint32_t    SINGLE_BUZZ = 0b10000000000000000000000000000000UL;
```

Comment:
Better static const, ins't it ?

Reply:
Only if we create more than one instance :-)

But yes, for clarity I've added `static`.